.github/workflows/ci+cd.yml

```diff
@@ -33,16 +33,18 @@ jobs:
         fetch-depth: 0 # fetch full history for GitVersion
 
     - name: Setup .NET
-      uses: actions/setup-dotnet@c0d4ad69d8bd405d234f1c9166d383b7a4f69ed8 # v2.1.0
+      uses: actions/setup-dotnet@v4.0.0
```

Comment:
For security reasons it's recommended to use the sha1 (which is immutable) instead of the tag (which can be changed in the repo). Probably/hopefully doesn't matter too much here for actions under GitHub's control, but...

Reply:
Done in commit 8fe98b56.